README.md

```diff
@@ -94,6 +94,15 @@ to your project.
 Now you can import the package like you would do like a normal npm dependency. Changes you will make 
 in your package will be seen in your repo.
 
+Example: linking the `asc-ui` package.
+
+```
+cd amsterdam-styled-components/packages/asc-ui
+yarn link
+cd ../../../atlas
```

Comment:
Developers that don't know `atlas` won't know what this means, I like the way this is explained in the npm docs with code and comments https://docs.npmjs.com/cli/link.html

Reply:
I pushed a new commit after reading docs.npmjs.com @royderks